src/js/form.js

```diff
@@ -13,6 +13,15 @@ const createTitle = () => {
 }
 // createElement('div', { className: 'form-group' })
 
+const setCurrentTime = input => {
+  const date = new Date();
+  const hour = date.getHours().toString().padStart(2, '0');
+  const minutes = date.getMinutes().toString().padStart(2, '0');
+  const time = hour + ':' + minutes;
+
+  input.value = time;
+}
```

Comment:
```suggestion
const setCurrentTime = input => {
  const fullDate = new Date();
  const time = fullDate.toLocaleTimeString('fr-FR', { hour: '2-digit', minute: '2-digit' });

  input.value = time;
}
```

Mais il serait plus propre de simplement retourner la valeur, et modifier l'input là où la fonction est appelée.

```suggestion
const getCurrentTime = () => {
  const date = new Date();
  return date.toLocaleTimeString('fr-FR', { hour: '2-digit', minute: '2-digit' });
}
```

avec plus loin :
```js
input.value = getCurrentTime();
// au lieu de: `setCurrentTime(input)`
```

Reply:
Bien vu pour le formatage, j'avoue que je ne connaissais pas ça 👍